libc/cmake/modules/CheckCompilerFeatures.cmake

```diff
@@ -42,16 +42,23 @@ set(AVAILABLE_COMPILER_FEATURES "")
 # Try compile a C file to check if flag is supported.
 set(CMAKE_TRY_COMPILE_TARGET_TYPE STATIC_LIBRARY)
 foreach(feature IN LISTS ALL_COMPILER_FEATURES)
+  set(compile_options ${LIBC_COMPILE_OPTIONS_NATIVE})
```

Comment:
should this be `compiler_options` instead of `compile_options`?

Reply:
Updated the below usage to the correct `compile_options`